api/v0/mfa_request_info.go

```diff
@@ -3,7 +3,7 @@ package v0
 // MfaRequestInfo struct returned for GetMfaRequest and ApproveMfaRequest
 type MfaRequestInfo struct {
 	// DateTime measured in seconds since unix epoch. A wrapper type for serialization that encodes a `SystemTime` as a `u64` representing the number of seconds since `SystemTime::UNIX_EPOCH`.
-	ExpiresAt int64 `json:"timestamp"`
+	ExpiresAt int64 `json:"expires_at"`
```

Comment:
I see `u64` in comment
```suggestion
	ExpiresAt uint64 `json:"expires_at"`
```